### DISCUSSION TITLE
modules -BlogUser , CmsUser ... - what is the relation with Users in main application  ?

### DISCUSSION THREAD

**Discussion by mrs01dev** [abpframework/abp#8648] [Q&A]
In modules  , a new user class is defined and inherit ( AggregateRoot<Guid>, IUser, IUpdateUserData)
In main application there is an AppUser.
What is the relation between users in main app and modules  ?
Does each modules   have separate users from main application , if true , how to arrange user login to the main app?

**Comment by ebicoglu** [comment]
`AppUser` entity shares the same table/collection ("AbpUsers" by default) with the IdentityUser entity of the Identity module.
The `AppUser` entity is used for to define your custom properties into this class.
You never create or delete this entity, because it is Identity module's job.
 You can query users from database with this entity.
 You can update values of your custom properties.
